### ISSUE TITLE
Downloader stops at episode 41

### ISSUE THREAD

**Issue by TenyamaGain** [hama3254/Crunchyroll-Downloader-v3.0#745]
For me, in any series I attempt to download with more than 41 episodes in one season, it completely just stops there, and I cannot download any further. Ordinarily, it'd show up on the UI, showing it's downloading X episode, but that stops after said episode number in said season. This even carries over after you reboot your downloader, for some reason. I'm going to give it a quick reinstall to double check, but this has happened in every situation I've tried.

**Comment by TenyamaGain** [comment]
Oh and if you try to download from 42 - X, it doesn't work there, either, unfortunately.

**Comment by TenyamaGain** [comment]
After a reinstall, it downloaded another 41 episodes on this series, but then just stopped oncemore at that cap of 41 episodes -- but it was because I reinstalled that it wasn't capped at 41, but rather, this time, 82.

**Comment by TenyamaGain** [comment]
In fact, the whole player stops functioning after that 41 mark -- I can't download anything else after that, until I reinstall.

**Comment by TenyamaGain** [comment]
And by "reinstall", I mean to replace all the files in the folder

**Comment by hama3254** [comment]
I don't see an reason why the Downloader would have to be reinstalled.
For a more detailed answer i would need to know what happens, where does the new downloads stop or fail.
If there is any kind of error message than I would need that and if not than a copy of the [CRD-Log](https://github.com/hama3254/Crunchyroll-Downloader-v3.0/wiki/CRD-Log) would be nice to have.

**Comment by TenyamaGain** [comment]
There's no error message, and it acts like it's functioning fine...except the main screen that houses the actual download bars doesn't keep adding onto it past that certain number. I'll look into getting you that log now, though.

**Comment by TenyamaGain** [comment]
I've tried closing the program then reopening -- nothing. Hell, even replacing all the files sometimes doesn't do it, sometimes I have to do  a clean folder install, and still it's jank. I've checked my firewall, it's not the issue, either.

**Comment by edimusxero** [comment]
I am testing it now to see, I know I downloaded from episode 42 on and it worked, but I may have deleted and re downloaded the program first, confirming now

**Comment by TenyamaGain** [comment]
Also, for the creator of this downloader -- few things that may help debug: When it starts to stop download, it says "loading" but nothing after that -- whereas when it's _really_ downloading, it says something like "Loading.." then "Downloading a3e3" or something like that -- that last step is absent when this occurs. Second thing is, Even if things are in the queue, if the thing is running (but of course not downloading, after the "breaking point"), you can exit and it has no warning that things're downloading -- that is completely absent, as well.

**Comment by TenyamaGain** [comment]
> I am testing it now to see, I know I downloaded from episode 42 on and it worked, but I may have deleted and re downloaded the program first, confirming now

Previous statement said, what're the results of this?

**Comment by Aiyomi-Irirako** [comment]
it's not the downloader. that's why he can't do anything about it
a new "install" is also not necessary

**Comment by TenyamaGain** [comment]
> 

You'd think, but only after a reinstall does it work -- I've tested it, and.. If I got to episode 41, waited 3-4 hours without doing anything after said event, come back, try again, it sitll won't work. Replacing it fixes it, however, no matter how little time has passed. So an install _is_ necessary. Though I do question whether it's a cache that needs to be cleared for their spam prevention, I could see that.

**Comment by Aiyomi-Irirako** [comment]
bro i doesnt think i know it
your results are wrong
test it correctly and you see it too

**Comment by hama3254** [comment]
For a somewhat useful answer i would need a full log and any additional error message that might pop-up.
I also want to add that i don't even try to reproduce the issue because 41 downloads per day is more than the downloader is meant to do.
I set the target to 12 episodes per day since this is what you could watch and the downloader is meant for private use only (based on §53 UrhG).

**Comment by edimusxero** [comment]
I can throttle it, I download 40 at a time, move the files out of the directory and close the program, open it back up and run it again and it works fine.  If it's a spam detection issue, I suppose I can live with that
